docs/tutorials/cmip6-recipe.ipynb

```diff
@@ -62,25 +62,25 @@
      "output_type": "stream",
```

Comment:
<p>"variabizes" 🤔</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>:sweat_smile: Even without the typo, that phrasing seemed a bit overwrought upon rereading. Changed it to:</p><p> </p><p>**Let's define a function that takes these strings as input**</p><p>- ... and returns full file paths!</p>